### PR TITLE
rpm: fix "rhel <= 7" conditional

### PR DESCRIPTION
**Pull request by smithfarm**
Fixes: https://tracker.ceph.com/issues/38810
Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

File: ceph.spec.in [hunk 1]
```diff
@@ -503,7 +503,7 @@ Requires:       python3-scipy
 Requires:       python2-scipy
 %endif
 %endif
-%if 0%{?rhel} <= 7
+%if 0%{?rhel} && 0%{?rhel} <= 7
```
Comment (tchaikov) [inline ceph.spec.in]: how about just put 
```
%if 0%{?rhel} && 0%{?rhel} == 7
```

as we don't build on RHEL6 or less. and this approach is also consistent with other places in this file.

**Comment by smithfarm** [comment]
jenkins test docs

**Comment by smithfarm** [comment]
jenkins test docs

**Comment by smithfarm** [comment]
https://shaman.ceph.com/builds/ceph/wip-38810/b0da2371c118445e5e32b748970eca0adbf7aa91/

**Pull request merged**